projects/plugins/crm/includes/ZeroBSCRM.DAL3.php

```diff
@@ -403,9 +403,7 @@ public function get_object_field_global( $object_type_id = -1 ){
 
         if ( $global_var_name !== -1){
 
-            global ${$global_var_name};
-
-            return ${$global_var_name};
+            return isset( $GLOBALS[ $global_var_name ] ) ? $GLOBALS[ $global_var_name ] : null;
```

Comment:
To avoid unexpected effects, I'd return `false` instead of `null`, like the default return below in this function.

The reason is that the old code relies on booleans and sometimes in `-1` value as if it were a `null` value.

Reply:
This code does not represent a change in the existing behavior. `global $foo`, if `$foo` is not set, results in `$foo` having a null value.